core/jobs/jobs_manager_test.py

```
@@ -310,3 +311,46 @@ def test_working_voiceover_sync_job(self) -> None:
+        for job_class, expected_id in self.TEST_CASES:
+            with (
+                mock.patch(
+                    'apache_beam.Pipeline',
+                    return_value=mock.MagicMock(),
+                ) as mocked_pipeline_class,
+                self.subTest(
+                    job_class=job_class,
+                    expected_id=expected_id,
+                ),
+            ):
+                jobs_manager.run_job(job_class, True, namespace=self.namespace)
```

Ruff (0.15.15)

[warning] Boolean positional value in function call

(FBT003)